### ISSUE TITLE
Disable vizion?

### ISSUE THREAD

**Issue by tehnorm** [Unitech/PM2#1074]
Vizion seems to be killing our system with vigorous `git show` and `git status` commands. Is there a way to disable this from happening - I don't care about PM2 to restarting the apps if the git repos change. This seems like an issue of either missing documentation or PM2 trying to be too damn smart for it's self.

```
top - 12:52:24 up  1:10,  2 users,  load average: 0.29, 0.15, 0.14
Tasks: 177 total,   2 running, 175 sleeping,   0 stopped,   0 zombie
%Cpu(s):  0.2 us,  1.4 sy,  0.0 ni, 98.4 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
KiB Mem:   2046484 total,  1905200 used,   141284 free,    55860 buffers
KiB Swap:   786428 total,        0 used,   786428 free.   911788 cached Mem

  PID USER      PR  NI    VIRT    RES    SHR S  %CPU %MEM     TIME+ COMMAND
13619 vagrant   20   0   19140   2040   1300 R   48.0  0.1   0:00.15 git
```

```
vagrant  13678  0.0  0.0   4444   652 ?        S    12:53   0:00 /bin/sh -c cd '/var/www/ijoin/queues/';LC_ALL=en_US.UTF-8 git status -s
vagrant  13674  0.0  0.0   4444   652 ?        S    12:53   0:00 /bin/sh -c cd '/var/www/ijoin/queues/';LC_ALL=en_US.UTF-8 git show --quiet --format=%H%n%aD%n%s%n%B HEAD
```

```
vagrant@dev:/var/www/ijoin/queues$ pm2 list
┌───────────────────────────┬────┬──────┬───────┬────────┬─────────┬────────┬─────────────┬──────────┐
│ App name                  │ id │ mode │ pid   │ status │ restart │ uptime │ memory      │ watching │
├───────────────────────────┼────┼──────┼───────┼────────┼─────────┼────────┼─────────────┼──────────┤
│ Simple Queue Broker       │ 0  │ fork │ 13399 │ online │ 0       │ 4m     │ 12.195 MB   │ disabled │
│ Enrollment PDF Consumer   │ 1  │ fork │ 13403 │ online │ 0       │ 4m     │ 40.531 MB   │ disabled │
│ Enrollment Email Consumer │ 2  │ fork │ 13409 │ online │ 0       │ 4m     │ 46.363 MB   │ disabled │
└───────────────────────────┴────┴──────┴───────┴────────┴─────────┴────────┴─────────────┴──────────┘
```

**Comment by jshkurti** [comment]
Ok, we'll implement an option so you can disable vizion completely at pm2 start.

**Comment by tehnorm** [comment]
@jshkurti thanks for the quick update. Is there anyway I can disable this for now - our dev boxes are getting murdered to the point of non responsiveness?

**Comment by jshkurti** [comment]
```$ npm install git://github.com/Unitech/pm2#no_vizion -g``` :)

**Comment by tehnorm** [comment]
@jshkurti 

![Image of Awesome](http://i.imgur.com/f2MK7zr.jpg)

**Comment by Unitech** [comment]
:+1:  :D

**Comment by jshkurti** [comment]
Haha <3
I'm closing the issue, I will comment here when the option ```--no-vizion``` is implemented and ready ;)

**Comment by tehnorm** [comment]
cool! thanks @jshkurti

**Comment by jshkurti** [comment]
Here we go :

```pm2 start app.js --no-vizion``` or ```"vizion" : false``` in the JSON declaration.

```npm i -g git://github.com/Unitech/PM2#development``` if you want to try right now, otherwise the official release is going to come tomorrow :)

**Comment by jshkurti** [comment]
It is published now in PM2 v0.12.11 ;)
https://github.com/Unitech/PM2/releases

**Comment by tehnorm** [comment]
@jshkurti thanks for the update! Looking forward to using the mainline release again.
